Description: affiche l'avertissement
 .
 liveusbcreator (0.1-2) unstable; urgency=medium
 .
   * ajout de la version de coreutils en dépendance
Author: arnault perret <arpinux@member.fsf.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- liveusbcreator-0.1.orig/liveUSBcreator
+++ liveusbcreator-0.1/liveUSBcreator
@@ -23,6 +23,7 @@ FAIL="An error occured"
 CHOICE="Choice"
 MODEL="Model"
 SIZEMSG="Size"
+CAREFUL="WARNING!\nThis tool will erase the entire selected disk.\n Choose wisely"
 
 case $LANG in
 	fr*)
@@ -36,6 +37,7 @@ case $LANG in
         CHOICE="Choix"
         MODEL="Modèle"
         SIZEMSG="Taille"
+        CAREFUL="ATTENTION!\nCet outil efface intégralement un disque pour y coller une ISO\nVeillez à bien choisir votre clé usb et non votre disque dur."
         ;;
         nl*)
         APPNAME="Handy live usb creator"
@@ -68,9 +70,13 @@ copytousb() {
     fi
 }
 
+zenity --info --title="LiveUSBcreator" --text="$CAREFUL"
 
 if [ -z "$1" ]; then
     ISO="$(zenity --title="$ISOMSG" --file-selection --file-filter="Disc Image *.iso | *.iso")"
+    if [ $? -ne 0 ]; then
+        exit 0
+    fi
 else
     ISO="$1"
 fi
@@ -94,7 +100,7 @@ if [ -z "$2" ]; then
         )
 
 
-    USB="/dev/$(zenity --width=350 --list  --text "Quel est le disque correspondant à la clé usb"  --radiolist \
+    USB="/dev/$(zenity --width=350 --list  --text "$USBMSG"  --radiolist \
         --column="$CHOICE" --column="USB-Key" --column="$MODEL" --column="$SIZEMSG" \
         $USBLIST | cut -d' ' -f2)"
 else
